src/components/pages/order/Main/Basket/BasketItems.js

```diff
@@ -1,23 +1,45 @@
 import React from "react";
 import styled from "styled-components";
-import { theme } from "../../../../../theme";
+import BasketCard from "./BasketCard";
+import { IMAGE_BY_DEFAULT } from "../../../../enums/product";
+
+export default function BasketItems({ basket, handleDeleteBasketItem }) {
+
+  const handleOnDelete = (id) => {
+    handleDeleteBasketItem(id)
+  }
 
-export default function BasketItems() {
   return (
-  <BasketItemsStyled>
-    Votre commande est vide.
-  </BasketItemsStyled>
-  )
+    <BasketItemsStyled>
+      {basket.map((item) => (
+        <div key={item.id} className="basket-card">
+          <BasketCard
+            {...item}
+            imageSource={item.imageSource ? item.imageSource : IMAGE_BY_DEFAULT}
+            onDelete={()=> handleOnDelete(item.id)}
```

Comment:
```suggestion
            onDelete={()=> handleDeleteBasketItem(item.id)}
```

Et donc tu pourras supprimer ton gestionnaire d'événement `handleOnDelete()`.